resources/views/admin/point/fields.blade.php

```diff
@@ -9,21 +9,21 @@
 <!-- Vote Field -->
 <div class="form-group col-sm-6">
     {!! Form::label('vote', trans('point.vote')) !!}
-    {!! Form::input('number', 'vote', 5 , ['class' => 'form-control', 'step' => '1', 'min' => '0', 'max' => '10']) !!}
+    {!! Form::input('number', 'vote', null , ['class' => 'form-control', 'step' => '1', 'min' => '0', 'max' => '10']) !!}
```

Comment:
[rank C] if you are repeating min & max input values, I think we should define it as constant in config.